internal/daysteps/daysteps.go

```diff
@@ -1,15 +1,45 @@
 package daysteps
 
 import (
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
 	"time"
+
+	"github.com/Pavel-Sergeev-ekb/first-project/tree/main/internal/spentcalories"
 )
 
 var (
 	StepLength = 0.65 // длина шага в метрах
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
-	// ваш код ниже
+	// Разделяем строку по запятой
+	parts := strings.Split(data, ",")
+
+	if len(parts) != 2 {
+		return 0, 0, errors.New("Некорректный формат данных")
+	}
+	// Парсим продолжительность
+	steps, err := strconv.Atoi(parts[0])
+	if err != nil {
+		return 0, 0, errors.New("Ошибка при парсинге шагов")
```

Comment:
1. Теряешь сообщение об ошибки
2. Ошибку лучше писать на английском
3. Ошибку лучше начинать с маленькой буквы
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```